assets/composer/settings.lando.php

```diff
@@ -0,0 +1,169 @@
+<?php
```

Comment:
Do we support Lando?

Reply:
We don't, but LGD does. That lando config stuff is there in intra's `settings.local.php`.

I'll take it out on both sites when I do the security updates.